### PR TITLE
Configure borgmatic periodicity

### PR DESCRIPTION
**Pull request by florentk**
Fix : 
 configure borgmatic periodicity #25
 make sure borgmatic commands don't overlap #26
 configure borgbase alertdays based on periodicity #27

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,12 @@ ENV KEEP_YEARLY="1"
 ENV DATABASES=""
 ENV MAX_BORGMATIC_RETRY="5"
 ENV WAIT_BEFORE_BORGMATIC_RETRY="5"
+ENV QUOTAS="2048"
```
Comment (rgaudin) [inline Dockerfile:15]: What are those for? Why are those not documented?

**Review by rgaudin — CHANGES_REQUESTED** [review]
Thank you ; works fine but there are still a few concerns:

- it's not as flexible as the doc. So we can't close the periodicity ticket with that iteration
- `quota` and `region` are setup option and should be cli args
- `PERIODICITY` is used in setup as well to define alert. This won't work as we don't export it in `setup-new-repo` so it would default to what's in Dockerfile… Or needs to be set which doesn't make sense. We should have an optional `--alert-days` to set it.
- all of those quota/region/alert should be documented. Both briefly in Dockerfile obviously but also in the documentation. 
- `quota` should be singular (not `quotas`)

If you prefer, open a ticket listing documentation updates in french and I'll update the documentation from time to time

Currently we can set incoherent values like `--every 45d` that would render as

```
12 3 */45 * * root /usr/bin/flock
```

This would raise no error but will not run the command either.

**Review by rgaudin — CHANGES_REQUESTED** [review]
Thanks ; still a couple of issues:

- `--alert-days` require a number of days with a suffix. Why is that??? It already has the unit in the arg name… and you're stripping it out anyway.
- Why do we have a quota by default? Makes no sense. Default should be no quota. Users can set quota if they want. Also, when suggesting a size for quota, it'd be adequate to use non-binary values (2000 instead of 2048) as that's what borgmatic and borgbase use.

**Comment by florentk** [comment]
> * `--alert-days` require a number of days with a suffix. Why is that??? It already has the unit in the arg name… and you're stripping it out anyway.
I wanted to make uniform with the backup command for perdiodicity, but we can remove the suffix.


> * Why do we have a quota by default? Makes no sense. Default should be no quota. Users can set quota if they want. Also, when suggesting a size for quota, it'd be adequate to use non-binary values (2000 instead of 2048) as that's what borgmatic and borgbase use.
Ok for no quota by default.

**Comment by rgaudin** [comment]
> > * `--alert-days` require a number of days with a suffix. Why is that??? It already has the unit in the arg name… and you're stripping it out anyway.
> 
> @rgaudin I wanted to make uniform with the backup command for perdiodicity, but we can remove the suffix.

I don't think it matters. periodicity is used on backup only and alert on setup only. If we were able to have an `--alert-every 1w` where we'd accept suffixes, that'd wise.
But since we're forced to days and because it's a different concept, I prefer we choose clarity over harmonization for this.

**Comment by florentk** [comment]
@rgaudin any news ?

**Comment by rgaudin** [comment]
Ah comme le premier changement visible c'est le `ENV PERIODICITY="1d"` dans le Dockerfile que tu as oublié de changé, je n'avais pas fait gaffe que tu avais changé les alertes.
Je regarde des que c'est changé

**Review by rgaudin — APPROVED** [review]
merci

**Pull request merged**